### ISSUE TITLE
Questions on how to improve model accuracy

### ISSUE THREAD

**Issue by bludger1** [AlexeyAB/darknet#3019]
Hi all,

First of all, thanks for the great work in this repo. I'm looking for some advice to train a (preferably) Tiny YoLo v3 network. As of know I've gathered a dataset consisting of roughly 6000 images all containing the objects where I'm interested in. This dataset consists of images not larger than 330x330px. The objects herein ranging from 100x100 px to roughly 30x30 in size and differ from color. Each image haves its own annotation file. Note that the images aren't all squares and are black and white.

As of now I'm testing the yolov3-tiny_3l configuration file. The reason is that I'd like to keep the footprint as small as possible.  In my real world application there will be multiple objects of the same class be very close to eachother in the same image. The goal is to count number of objects and give the most precise location of the objects. Also there will be no GPU available in the application so a small network is preferable.

To further improve the current network, I have some questions and seek for some advice:

- What is the prefered size configuration of the YoLo network in my case?
- As there isn't much difference in backgrounds across my dataset it is might not ideal. Would it help to change the background (eg. area outside
the bouding box) to another random color? (In case YoLo takes this area into account)
- The objects differ in color and size but are still the same thing in the end, is it recommend to have just one class or multiple?
- Would it be helpful to recalculate anchors and/or change the masks in this case?
- Would it still help to use a pretrained model? The objects I want to detect aren't trained by any of the pretrained models.

With my current configuration I get an mAP of about 80-90% with an average loss of 0.2. I consider the loss too high as it misses still quite a lot
of objects during tests. Thereby I assume that the mAP can be better aswell.

Thanks!
